ports/esp32/boards/sdkconfig.ble

```diff
@@ -17,4 +17,4 @@ CONFIG_BT_NIMBLE_PINNED_TO_CORE=1
 
 # Increase NimBLE task stack size from the default, because Python code
 # (BLE IRQ handlers) will most likely run on this task.
-CONFIG_BT_NIMBLE_TASK_STACK_SIZE=6144
+CONFIG_BT_NIMBLE_TASK_STACK_SIZE=4096
```

Comment:
This change probably should go in a chip-specific `sdkconfig.c2`, unless we're very sure all SoCs can run with the smaller stack size?

(The git log shows it had to be increased in cd66aa05c so there is some risk of regression if we make it smaller on all chips.)